pandas/tests/test_generic.py

```diff
@@ -1497,6 +1497,40 @@ def test_to_xarray(self):
                            expected,
                            check_index_type=False)
 
+    def test_filter(self):
```

Comment:
There is already a `test_filter` in https://github.com/pydata/pandas/blob/c2ea8fb20611370c7db7e11506727b0b67a76662/pandas/tests/frame/test_axis_select_reindex.py#L621, maybe good to add this there?